UPGRADE.rst

```diff
@@ -88,6 +88,14 @@ for example:
 Upgrading to v1.32.0
 ====================
 
+Dropping support for end-of-life Python, Postgres and SQLite versions
```

Comment:
This is the most tiny nitpick, but I don't think minor versions of SQLite actually go EOL / have a defined support lifetimes.

Maybe just `s/end-of-life/old/` on the heading?